buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersions.kt

```diff
@@ -39,5 +41,11 @@ object IntellijVersions {
     IntellijVersionHelper.getMajorPart(latestStable)
   }
 
-  val buildTarget: String = eapOfLatestSupportedMajor ?: latestStable
+  // This allows to change the target IntelliJ version
+  // by using a project property 'overrideBuildTarget' while running tasks like runIde
+  val buildTarget: String = overrideBuildTarget ?: eapOfLatestSupportedMajor ?: latestStable
+
+  override fun getPublicType(): TypeOf<*> {
```

Comment:
Out of curiosity... does Gradle require it for all extensions? 🤔

Reply:
Well, turns out it's not needed if the public type you want the extension to be available as is the same as the class type. Hope I understand it correctly. This is connected to the [ExtensionAware interface](https://gradle.github.io/kotlin-dsl-docs/api/org.gradle.api.plugins/-extension-aware/index.html).

Reply:
[Extension Container API](https://gradle.github.io/kotlin-dsl-docs/api/org.gradle.api.plugins/-extension-container/index.html) also some info about the methods here.